src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -52,102 +28,28 @@ def post(self, request: Request) -> Response:
         :pparam: string dummyEvent: pass 'True' to load a dummy event instead of providing one in the request
         :pparam: string dummyOccurrence: pass 'True' to load a dummy occurrence instead of providing one in the request
         """
-        event = {}
-        if request.query_params.get("dummyEvent") == "True":
+        if request.query_params.get("dummyData") == "True":
```

Comment:
If the param is changing you should update the docstring, but also I am semi-against changing the param cause then anyone using it needs to pass different stuff. I realize it's a small amount of people since it's an internal endpoint but still 🤷

Reply:
How about I just use `dummyOccurrence`?  I don't think there's too much value in having both any more, since they're being loaded from the same place